### ISSUE TITLE
Theme does not work in emacsclient?

### ISSUE THREAD

**Issue by guilhermeasg** [greduan/emacs-theme-gruvbox#36]
I'm really not sure about this one as I've never dug very deep into Elisp, so please confront me if I'm saying something silly.

This may have something to do with [this SO question](http://stackoverflow.com/questions/15555309/emacs-for-windows-error-loading-color-theme), I believe.
## Steps to reproduce
1. Kill Emacs daemon (if it was already running)
2. Launch Emacs (`emacs` from the command line)
3. `M-x customize-themes RET`
4. Select `gruvbox` and "Save theme settings"
5. Close emacs
6. Launch Emacs client and server (`emacsclient -nc -a ""` from command line)

<details>
  <summary>

Output from `*Messages*`</summary>

```
Unable to load color "color-235"
Unable to load color "color-250"
Unable to load color "color-239" [2 times]
Unable to load color "color-235" [2 times]
Unable to load color "color-175"
Unable to load color "color-235" [3 times]
Unable to load color "color-239"
Unable to load color "color-250"
Unable to load color "color-237"
Unable to load color "color-246"
Unable to load color "color-235" [6 times]
Unable to load color "color-239"
Unable to load color "color-235" [5 times]
Unable to load color "color-239"
Unable to load color "color-250"
Unable to load color "color-237"
Unable to load color "color-246"
Unable to load color "color-235" [6 times]
Unable to load color "color-239"
Unable to load color "color-235" [4 times]
Unable to load color "color-243"
Unable to load color "color-239"
Unable to load color "color-250"
Unable to load color "color-235"
Unable to load color "color-250"
Unable to load color "color-239" [2 times]
Unable to load color "color-235" [2 times]
Unable to load color "color-175"
Unable to load color "color-235" [2 times]
Unable to load color "color-239"
Unable to load color "color-237"
Unable to load color "color-235" [4 times]
Unable to load color "color-239"
Unable to load color "color-250"
Unable to load color "color-237"
Unable to load color "color-246"
```

</summary>

**Comment by jasonm23** [comment]
Due to the way the Gruvbox theme is built you cannot use customize-theme with it.

**Comment by guilhermeasg** [comment]
Interesting, but the same happens if I load it via `load-theme` on `init.el`. The problem seems to be in the order in which components are initialized.

**Comment by jasonm23** [comment]
How did you install it?  If you've done a manual install remove it and install via MELPA.

**Comment by guilhermeasg** [comment]
It had already been installed from MELPA (`20160514.658`). I reinstalled it and this is still happening. This is odd because it seems not to happen with other themes. Are you not able to reproduce it as well?

**Comment by jasonm23** [comment]
I cannot reproduce it

**Comment by guilhermeasg** [comment]
Welp. Gotta be something with my own config then. Thanks for your attention!

**Comment by jasonm23** [comment]
Try renaming your ~/.emacs.d temporarily and start Emacs again, and re-install GruvBox.

Please also make a note of your Emacs version and build parameters.

**Comment by jasonm23** [comment]
After running that test, simply remove the newly created .emacs.d and rename/move your original back in place.

**Comment by jasonm23** [comment]
(By the way, please note I am Slomojo on that Stackoverflow question.)

**Comment by guilhermeasg** [comment]
> Try renaming your ~/.emacs.d temporarily and start Emacs again, and re-install GruvBox.
> 
> Please also make a note of your Emacs version and build parameters.

Same result: works for Emacs, but not `emacsclient`.
<details>
<summary>*Messages* buffer</summary>
```
Loading /usr/share/emacs/site-lisp/site-start.d/desktop-entry-mode-init.el (source)...done
Loading /usr/share/emacs/site-lisp/site-start.d/git-init.el (source)...done
Starting Emacs daemon.
Unable to load color "color-235"
Unable to load color "color-239"
Unable to load color "color-250"
Unable to load color "color-237"
Unable to load color "color-246"
Unable to load color "color-235" [6 times]
Unable to load color "color-239"
Unable to load color "color-235" [4 times]
Unable to load color "color-243"
Unable to load color "color-239"
Unable to load color "color-250"
When done with this frame, type C-x 5 0
Unable to load color "color-235"
```
</details>

> (By the way, please note I am Slomojo on that Stackoverflow question.)

Weirdest internet coincidence I found in quite some time. :stuck_out_tongue:

**Comment by jasonm23** [comment]
e1c1564c2a95388f2dbd92870e2caa3562927aa6 should address this problem.

**Comment by jasonm23** [comment]
(released as 1.00)
